### PR TITLE
Give a name and icon for slackbot.

### PR DESCRIPTION
**Pull request by bazimov**

### CODE REVIEW

File: src/foremast/utils/slack.py [hunk 1]
```diff
@@ -35,7 +35,7 @@ def post_slack_message(message, channel):
     LOG.debug('Slack Channel: %s\nSlack Message: %s', channel, message)
     slack = slacker.Slacker(SLACK_TOKEN)
     try:
-        slack.chat.post_message(channel, message)
+        slack.chat.post_message(channel=channel, text=message, username='pipeline-bot', icon_emoji=':gear:')
```
Comment (e4r7hbug) [inline src/foremast/utils/slack.py]: Make `username` and `icon_emoji` arguments for `post_slack_message()` so we, or other people, can change a little more easily.

Comment (bazimov) [inline src/foremast/utils/slack.py]: I have added them as arguments now.

**Comment by bazimov** [comment]
@e4r7hbug yes missed those static args, now corrected.

**Comment by e4r7hbug** [comment]
Looks good for merging.

**Pull request merged**